### PR TITLE
add document about combination_rules 

### LINKED ISSUE #174: Document combination fv_converter (combination_rules)
https://github.com/jubatus/jubatus_core/pull/104

### PR DESCRIPTION
**Pull request by shiodat**
fix #174

### CODE REVIEW

File: source/en/fv_convert.rst [hunk 1]
```diff
@@ -575,6 +575,71 @@ It depends on "type" how to specify weight and name features.
  :type:   Specifies the name of extractor in use. The extractor is either one defined in "binary_types". Note that no pre-defined extractors are prepared.
 
 
+Feature Extraction from Combination Data
+-------------------------------
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: The underline is too short (will be warned by Sphinx)

File: source/en/fv_convert.rst [hunk 2]
```diff
@@ -575,6 +575,71 @@ It depends on "type" how to specify weight and name features.
...
+
+We can make new combination features by combining number features oｒ string features.
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: Contaminated by `ｒ` full-width char.

File: source/en/fv_convert.rst [hunk 3]
```diff
@@ -575,6 +575,71 @@ It depends on "type" how to specify weight and name features.
...
 
 
+Feature Extraction from Combination Data
+-------------------------------
+
+We can make new combination features by combining number features oｒ string features. 
+As with strings and numbers, feature extraction rules are also described for combination types.
+We can make user-defined extractors for combination types, too.
+
+.. code-block:: js
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: It is better to describe the meaning of this example configuration, like in the Japanese document.

File: source/en/fv_convert.rst (hunk 1 above)
Comment (shiodat) [inline source/en/fv_convert.rst]: Fix the underline that has same length of the title.

File: source/en/fv_convert.rst (hunk 2 above)
Comment (shiodat) [inline source/en/fv_convert.rst]: Replace the character.

File: source/en/fv_convert.rst (hunk 3 above)
Comment (shiodat) [inline source/en/fv_convert.rst]: I added the description like Japanese docs.

**Comment by shiodat** [comment]
@kmaehashi Thanks for review!
I fixed the underline, char and added description of combination rules.

File: source/en/fv_convert.rst [hunk 4]
```diff
@@ -575,6 +575,83 @@ It depends on "type" how to specify weight and name features.
...
+      ],
+      "combination_types": {},
+      "combination_rules": [
+        { "key_left": "*@num", "key_right": "*@num", "type": "add"},
+        { "key_left": "*@num", "key_right": "*@num", "type": "mul"},
+        { "key_left": "*@str#bin/bin", "key_right": "*@str#bin/bin", "type": "add"}
+        { "key_left": "*@str#bin/bin", "key_right": "*@str#bin/bin", "type": "mul"}
+      ]
+
+      If you want to combinate all string features, you can
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: Unneeded sentence?

Comment (shiodat) [inline source/en/fv_convert.rst]: Sorry, I remove the sentense.

**Comment by kmaehashi** [comment]
:+1: Thanks!

**Pull request merged**